bin/remove_it.py

```diff
@@ -231,7 +246,7 @@ def clean_section(pub, section, conf, is_dry_run=True):
             kws[key] = int(info[key])
         except KeyError:
             pass
-    ref_time = datetime.utcnow() - timedelta(**kws)
+    ref_time = dt.datetime.now(dt.UTC) - dt.timedelta(**kws)
```

Comment:
`dt.UTC` is added in 3.11 so it won't work with Python 3.10 which we support. Until we drop 3.10 we should use `dt.timezone.utc` instead.

Reply:
Yep, fixed